src/OrbitVulkanLayer/SubmissionTracker.h

```diff
@@ -525,18 +527,28 @@ class SubmissionTracker : public VulkanLayerProducer::CaptureStatusListener {
       return;
     }
 
-    if (!queue_to_submissions_.contains(queue)) {
-      queue_to_submissions_[queue] = {};
+    if (!submissions_queue_.contains(queue)) {
+      std::priority_queue<QueueSubmission, std::vector<QueueSubmission>,
+                          std::function<bool(QueueSubmission, QueueSubmission)>>
+          priority_queue{[](const QueueSubmission& lhs, const QueueSubmission& rhs) -> bool {
+            return lhs.meta_information.pre_submission_cpu_timestamp >
+                   rhs.meta_information.pre_submission_cpu_timestamp;
+          }};
```

Comment:
I suggest defining the comparator once as `static constexpr` right before the definition of `submissions_queue_`.

Reply:
Then you might (maybe) even be able to just write `submissions_queue_.emplace(queue, {kTheNameYouGaveTheComparator}`.

Reply:
Nice, done!